README.md

```diff
@@ -1,23 +1,23 @@
 # Č
 
-The czech library for C. Why? Because we could, of course!
+The Czech library for C. Why? Because we could, of course!
 
 Please contribute your other ~~horrible~~ awesome and creative translations
 so that we can write a fully czech code to suit all our needs.
 
 ## Using
 
 This project resides solely in a single header file, aptly named `czech.h`.
-Download that file alone, or along this whole repository, and then, after
-including it in your source files like you would with any other header, you
-will be fully set up! (Just make sure you use relatively modern compiler)
+Download that file alone, or the entire repository, and then, after
+including it in your source files like you would any other header, you'll
+be good to go! (Just make sure to use a relatively modern compiler)
 
-We have this project tested and provably working with
+This project has been tested with and works for
 [Clang](http://releases.llvm.org/download.html), but newer versions of gcc
 are also known to work, as well as some other compilers.
 
 ## Integrated transpiler
-For testing, we also provide C to Č and reverse transpiler in the file
+For testing, we also provide C to Č and a reverse transpiler in the file
```

Comment:
Here I think the article suggests that we provide "C to Č" and "a reverse transpiler" instead of "C to Č and reverse" - transpiler. Do you see it the same way? 

Perhaps there are better ways to phrase this like 
`we provide a transpiler for both C to Č and Č to C`
`we provide a transpiler between C and Č that works both ways`
Idk